src/routes/users/registerLogin.js

```diff
@@ -263,7 +263,8 @@ router.post(
         }
       });
 
-      if (Object.keys(updateObj).length === 0) {
+      if (!Object.keys(updateObj).length) {
+        res.statusMessage = 'INVALID_PAYLOAD';
         res.sendStatus(400);
```

Comment:
In express 4.0 we can write
```suggestion
        res.status(400).send('INVALID_PAYLOAD');
```

Reply:
Got it~! I was wondering if I should follow the coding style of the rest of code.

Reply:
We use `sendStatus` if we don't want to modify the message